kalite/distributed/management/commands/setup.py

```diff
@@ -96,6 +96,25 @@ def get_hostname_and_description(hostname=None, description=None):
     return (hostname, description)
 
 
+def get_assessment_items_filename():
+    def validate_fn(fn):
+        try:
+            open(fn, "r").close()
+            return True
+        except IOError:
+            sys.stderr.write("Error: couldn't open the specified file: \"%s\"\n" % fn)
+            return False
+
+    prompt = "Please enter the filename of the assessment items package you have downloaded (leave blank to skip this step): "
```

Comment:
Would be good to allow this to be a full path (and ask them for it), in case they downloaded it to elsewhere. But still allow relative/filename-only entries.